spec/facility_spec.rb

```diff
@@ -1,26 +1,268 @@
-  end
+    let (:facility_1) { Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'}) }
+    let (:facility_2) { Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'}) }
+    let (:cruz) { Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice}) }
+    let (:bolt) { Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev}) }
+    let (:camaro) { Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice}) }
+    let (:registrant_1) { Registrant.new('Bruce', 18, true ) }
+    let (:registrant_2) { Registrant.new('Penny', 16) }
+    let (:registrant_3) { Registrant.new('Tucker', 15) }
+
   describe '#initialize' do
     it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
-      expect(@facility.name).to eq('DMV Tremont Branch')
-      expect(@facility.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
-      expect(@facility.phone).to eq('(720) 865-4600')
-      expect(@facility.services).to eq([])
+      expect(facility_1).to be_an_instance_of(Facility)
+      expect(facility_1.name).to eq('DMV Tremont Branch')
+      expect(facility_1.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(facility_1.phone).to eq('(720) 865-4600')
+      expect(facility_1.hours).to eq(nil)
+      expect(facility_1.holidays_closed).to eq(nil)
+      expect(facility_1.services).to eq([])
+      expect(facility_1.registered_vehicles).to eq([])
+      expect(facility_1.collected_fees).to eq(0)
     end
   end
 
-  describe '#add service' do
+  describe '#services' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(facility_1.services).to eq([])
+      facility_1.add_service('New Drivers License')
+      facility_1.add_service('Renew Drivers License')
+      facility_1.add_service('Vehicle Registration')
+      expect(facility_1.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end
+  end
+
+  describe '#has_service?' do
+    it 'can check if it has a service' do
+      expect(facility_1.services).to be_empty
+
+      facility_1.add_service('Written Test')
+      facility_1.add_service('New Drivers License')
+
+      expect(facility_1.services).to eq(['Written Test', 'New Drivers License'])
+
+      expect(facility_1.has_service?('Written Test')).to be true
```

Comment:
This it block is actually testing 3 methods. add_service, services, and has_service? We want to try to practice single responsibility for our tests as well. Each test should get it's own it block.